rotkehlchen/db/utils.py

```diff
@@ -151,27 +151,27 @@ class DBAssetBalance(NamedTuple):
     category: BalanceType
     time: Timestamp
     asset: Asset
-    amount: str
-    usd_value: str
+    amount: FVal
+    usd_value: FVal
 
     def serialize(
             self,
             export_data: Optional[Tuple[Asset, Price]] = None,
-    ) -> Dict[str, Union[str, int]]:
+    ) -> Dict[str, Union[str, int, FVal]]:
```

Comment:
Please add a docstring to this function. I have had a hard time to understand what `export_data` is and what it does.

Reply:
This is still not fixed. You wrote:

```
If `export_data` is provided, the data is serialized for human consumption i.e
`usd_value` is converted to the user's currency & timestamp is converted to a readable one.
```

I still don't understand what is `export_data`. What are the possible values for it?